packages/notify-client/src/controllers/engine.ts

```diff
@@ -922,13 +922,13 @@ export class NotifyEngine extends INotifyEngine {
             "notify_get_notifications_response"
           );
 
-        const mappedNotifications: NotifyClientTypes.NotifyMessage[] =
+        const mappedNotifications: NotifyClientTypes.NotifyNotification[] =
           claims.nfs.map((nf) => ({
             body: nf.body,
             id: nf.id,
             sentAt: nf.sent_at,
             title: nf.title,
-            url: nf.url,
+            url: nf.url ?? null,
```

Comment:
`??` doesn't work because `url` would be `""` not `null`, so this is basically a no-op currently. Needs to be `||` or better a `?:` to be more explicit about what's going on:

```suggestion
            url: nf.url != "" ? nf.url : null,
```

Reply:
Ah I understood it the other way around